build.gradle

```diff
@@ -85,15 +85,20 @@ allprojects {
 
 dependencies {
   api "org.wiremock:wiremock:$versions.wiremock"
-
-  implementation 'javax.annotation:javax.annotation-api:1.3.2'
+  //  1.x is not maintained but we need to use to support Java 11. It is in a better state than java-faker, which is also not maintained. Bump to 2.x once we deprecate support for Java 11.
```

Comment:
It is not going to happen anytime soon, at least on the WireMock core level. Nothing prevents from doing Java 17 only extensions if documented properly

Reply:
Is it safe to use Java 17 extensions in a Java 11 project? I think we're just being cautious here to make sure there's no Java 17 specific constructs being used in data-faker.